src/components/Title/Title.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
```

Comment:
Overkill

Reply:
doesn't work without the React import. Should i rather just import from a single location like the other imports?

Reply:
I dont think this has been addressed - we need to delete this file and just use h2 tags somewhere else

Reply:
Remove this component and integrate to where its called.